### ISSUE TITLE
Disable Cmd-W and Cmd-Q on the UTM window

### ISSUE THREAD

**Issue by jy** [utmapp/UTM#5513]
**Describe the issue**  

Cmd-W and Cmd-Q both go to the UTM to close the window or quit UTM.

Both of these are used very frequently in my guest OS and I don't really use or need them to close or quit UTM.

Capturing the mouse works, but it is annoying to have to do as I work in both host and guest simultaneously.

It would be great to have an option to disable Cmd-W and Cmd-Q on the UTM window so they pass through to the guest.

**Configuration**  
* UTM Version: 4.2.5
* macOS Version: 13.4.1
* Mac Chip (Intel, M1, ...): Intel

**Comment by theosp** [comment]
Thanks to the discussion here: https://arstechnica.com/civis/threads/disable-cmd-w-for-a-specific-application.1166730/ | https://archive.is/wip/Lfmmx .

> System Preferences > Keyboard pane > Keyboard Shortcuts tab > Application Shortcuts option in the left-hand list > Click on "+" button beneath > Follow the instructions

So, you just need to pick a disabled option in the menu.

For example, I picked "Show Sidebar"

<img width="413" alt="2024-02-10_16-01-41" src="https://github.com/utmapp/UTM/assets/207494/562e5729-a592-4458-ae1c-b19dcfda2445">

Then you set:

<img width="715" alt="2024-02-10_16-02-40" src="https://github.com/utmapp/UTM/assets/207494/8bdd892c-19f3-4da2-9f43-fd3df858a24f">

The Command W now literally closing tabs on my firefox - as intended. It works perfectly.

**Comment by lloeki** [comment]
> especially not without asking

I think that's one of the root causes here: whatever way a close is triggered (keyboard shortcut, menu entry, titlebar "close" button) *it should not close without confirmation*, especially as *closing is equivalent to yanking the plug* and not sending any ACPI shutdown event or something.

Note that `Capture input devices` is a good workaround as well: any and all keyboard event is going to the VM, including the mentioned ones. Sadly *there's no setting to make it a default* so you have to remember to click it on the toolbar.

**Comment by PierrotAWB** [comment]
@dawadaq Notice that in settings, you can specify the application for which your shortcut applies. So, if you set application to UTM, then only UTM is affected. 

theosp's comment isn't explicit about this, but when you take some key combination, say, ⌘W, that's already assigned to something (Close), and you bind it to something else, the old binding is removed. By making that 'something else' an extent and _disabled_ menu option (in his case, Show Sidebar), you free up ⌘W:

<img width="349" alt="Screenshot 2024-10-28 at 3 59 22 PM" src="https://github.com/user-attachments/assets/c05e5c37-6550-4271-bb51-8baba0ea6f69">


So, the following, while hacky, does precisely what I want:

<img width="454" alt="Screenshot 2024-10-28 at 4 00 49 PM" src="https://github.com/user-attachments/assets/a81ee4fb-839d-4f2f-b47a-9e5d81c4712a">

and leaves behaviour in every other application on my system intact.

**Comment by a2379** [comment]
Bumping - this is still a very annoying issue that conflicts with keyboard shortcuts inside the VM. It appears to be tagged for release 4.4 which was a long time ago, but is still not fixed as of the current release.

**Comment by impishlucy** [comment]
The next issue that is still open and heavily annoying. Does any dev here even listen to what the community wants ?

**Comment by theosp** [comment]
In our sci-fi present... I'd simply ask a strong agent to take care of it.

**Comment by impishlucy** [comment]
Yeah suggesting Slop AI is defenetly an answer, a shit one, but atleast its one.
